src/metrics/scenario.jl

```diff
@@ -60,19 +64,38 @@ end
 scenario_relative_cover = Metric(_scenario_relative_cover, (:timesteps, :scenarios))
 
 """
-    scenario_relative_juveniles(data::YAXArray, coral_spec::DataFrame, k_area::AbstractVector{<:Real}; kwargs...)::AbstractArray{<:Real}
+    scenario_relative_juveniles(data::YAXArray{<:Real,3}, coral_spec::DataFrame, k_area::AbstractVector{<:Real}; kwargs...)::AbstractArray{<:Real}
```

Comment:
You've renamed `data` to `X` in the implementation but it is left as is here.

Reply:
Done